examples/preflight/sample-preflight.yaml

```diff
@@ -60,13 +60,18 @@ spec:
           - pass:
               when: "== aks"
               message: AKS is a supported distribution
-          # Will be supported in the future
           - pass:
               when: "== kurl"
```

Comment:
@DanStough My assumption is that the `pass` logic stops at the first one that passes, in which case, if rke2 is installed by kURL, this preflight will always return kURL, even if it is also rke2. Is that your intended behavior? If not, would it make sense to put the more-specific distros above the kurl check?

Reply:
That is how I understand it works. My thought was somehow that kurl is a more specific/interesting case than rke or k3s alone. The vendor would define the kurl spec, so this would be known and higher priority, than roll your own k3s or rke2. I see it works the other way though. I'm happy to change it if you think it's the other way around.

Reply:
That's fine. I wanted to make sure you thought about it and it was intentional. It may cause confusion for someone, but the vendor can certainly change the order if they care more about that part.